src/ApiGuardian.php

```diff
@@ -53,11 +53,20 @@ public function __invoke(ApplicationInterface $app)
         /** @var Request $request */
         $request = $app->getRequest();
 
-        if (!$request->hasHeader('Authorization')) {
+        // Fix for issue #1
+        $httpHeader = [];
+        if (\function_exists('apache_request_headers')) {
+            $httpHeader = apache_request_headers();
+        }
+
+        if (!isset($httpHeader['authorization']) && !$request->hasHeader('Authorization')) {
```

Comment:
It should be 'Authorization' instead of 'authorization'.

Reply:
no, it's `authorization` `apache_request_headers()` removes all case